### DISCUSSION TITLE
How to change tabulation?

### DISCUSSION THREAD

**Discussion by rokokol** [codewars/codewars.com#2952] [Q&A] [answered]
Hello! Could someone help me change indent? It is one space symbol, however I want increase it to 4 ones 🙏

**Comment by hobovsky** [comment, marked as answer]
It is possible to change tabulation settings for kata trainer in general, but there is no GUI for this. You need to change some options with developer tools in your browser: https://github.com/codewars/codewars.com/issues/2378#issuecomment-838138021

**Reply by rokokol** [reply]
Thank you
